core/src/SplitView.html

```diff
@@ -307,8 +307,9 @@ <h1 class="fd-splitView__title">{splitViewSettings.title}</h1>
 
 <style type="text/scss">
   $topNavHeight: 44px;
-  $leftNavWidth: 320px;
+  $leftNavWidth: 15em;
   $collapsedSplitviewHeight: 38px;
+  $collapsedSplitViewContainerHeight: 49px;
```

Comment:
I would recommend removing these changes with height and instead, just add `margin-top: 0;` to the `fd-splitView__title`